### ISSUE TITLE
Classification performance of provided BiT checkpoint is unexpected

### ISSUE THREAD

**Issue by BierOne** [deeplearning-wisc/gradnorm_ood#11]
Hi,

I sincerely appreciate your public code. I can reproduce your OOD detection scores by using the provided BiT checkpoint. But I found that the classification performance of this checkpoint is quite strange (about 0.1 on ImageNet val).

For your information, I also validate other models on the ImageNet-1k validation set. E.g., if I utilize the pretrained ResNet-50 (from torchvision), the performance is in the regular range (about 80% ACC).

Could you please check your provided ckt? Thank you so much.

**Comment by iurgnauh** [comment]
Hi,

Thanks for being interested in our work! I guess the label number you are using is different from the model? Can you try the ground-truth label in this codebase: https://github.com/deeplearning-wisc/large_scale_ood/blob/master/data_lists/imagenet2012_val_list.txt?

The model we used in our paper is slightly different from the original BiT checkpoint. Even though GradNorm doesn't require any extra training, for easy comparison to all baselines, we reused a model in our previous paper, with some slight fine-tuning from BiT, as you can see in the other codebase.

**Comment by BierOne** [comment]
Thanks! It solves the problem. After using the provided image labels, the model works well.
